core/src/main/java/com/microsoft/applicationinsights/internal/quickpulse/QuickPulse.java

```diff
@@ -44,6 +44,7 @@
 public enum QuickPulse implements Stoppable {
     INSTANCE;
 
+    public static final Integer QP_INVARIANT_VERSION = 1;
```

Comment:
it's more conventional to make this a primitive
```suggestion
    public static final int QP_INVARIANT_VERSION = 1;
```